### PR TITLE
Update to alpine 3.8

### PR DESCRIPTION
**Pull request by RX14**

### CODE REVIEW

File: linux/Dockerfile [hunk 1]
```diff
@@ -55,8 +55,8 @@ FROM ${alpine_image}
 RUN apk add --no-cache \
       # Statically-compiled llvm
       llvm4-dev llvm4-static \
```
Comment (j8r) [inline linux/Dockerfile]: You can use llvm5

Comment (RX14) [reply linux/Dockerfile]: That's a change for another PR.

File: linux/Dockerfile [hunk 2]
```diff
@@ -76,16 +76,6 @@ RUN git clone https://github.com/ivmai/bdwgc \
...
-ARG libevent_version
-RUN git clone https://github.com/libevent/libevent \
- && cd libevent \
- && git checkout ${libevent_version} \
- \
- && ./autogen.sh \
- && ./configure --disable-debug --disable-shared \
- && make -j$(nproc)
-
 ENV LIBRARY_PATH=/bdwgc/.libs/:/libevent/.libs/
```
Comment (Sija) [inline linux/Dockerfile]: Shouldn't `/libevent/.libs/` be dropped from `LIBRARY_PATH`?

Comment (RX14) [reply linux/Dockerfile]: send a PR pls

Comment (Sija) [reply linux/Dockerfile]: done, see #25

**Pull request merged**